javatron/src/main/java/com/newrelic/api/Behaviors/InvalidQueryBehavior.java

```diff
@@ -0,0 +1,29 @@
+package com.newrelic.api.Behaviors;
+
+import java.sql.SQLException;
+
+import com.newrelic.lib.ApplicationContainer;
+import com.newrelic.lib.Inventory.MySQLRepository;
+
+public class InvalidQueryBehavior extends Behavior
+{
+    public static final String Name = "INVALID-QUERY";
+
+    public InvalidQueryBehavior()
+    {
+        this.setName(InvalidQueryBehavior.Name);
+    }
+
+    public void Execute() throws SQLException, Exception
+    {
+        super.Execute();
+        var appConfig = ApplicationContainer.getInstance().GetAppConfigRepository();
+        var mySQLConfig = appConfig.FindMySQLConfiguration();
+        if (mySQLConfig != null)
+        {
+            var repo = MySQLRepository.getInstance(mySQLConfig);
```

Comment:
can this be injected into the class? 🙈 
if not, or the lift seems too difficult, feel free to ignore this.

Reply:
Absolutely! I was still on my weird logic from Nodetron, updating now...